certified-connectors/Datablend/readme.MD

```diff
@@ -0,0 +1,35 @@
+# Datablend Connector
+Datablend is a powerful iPaaS solution with a focus is serving the needs of CFOs, controllers, and financial planning and analysis (budgeting) professionals as they look to streamline processes, speed up closes, and increase confidence in reporting.
```

Comment:
@bmp02050 please check the text for typographical errors once more.
_Datablend is a powerful iPaaS solution with a focus_ **is** _serving the needs of ..._

Reply:
changed is to on